packages/manager/modules/pci/src/projects/project/serving/namespace/infos/detach-registry/detach.module.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import routing from './detach.routing';
+import component from './detach.component';
+
+import registry from '../registry/registry.module';
+
+const moduleName = 'ovhManagerPciProjectServingNamespaceInfosDetachRegistry';
+
+angular.module(moduleName, [
+  'ngTranslateAsyncLoader',
+  'oui',
+  'ovh-api-services',
+  'pascalprecht.translate',
+  'ui.router',
+  registry,
+])
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .component('ovhManagerPciProjectServingNamespaceInfosDetachRegistryComponent', component);
```

Comment:
Name is quite long, maybe it could be shorten a little
```suggestion
  .component('pciProjectServingNamespaceInfosDetachRegistry', component);
```